src/components/Nav/NavUtils.tsx

```diff
@@ -11,11 +11,12 @@ const buildCommonQueryParams = (params: GraphParamsType): string => {
 };
 
 export const makeNamespaceGraphUrlFromParams = (params: GraphParamsType): string => {
```

Comment:
maybe rename to makeNamespacesGraphUrlFromParams

Reply:
+1